linux_os/guide/system/auditing/configure_auditd_data_retention/var_audispd_network_failure_action.var

```diff
@@ -10,9 +10,10 @@ interactive: false
 
 options:
     default: single
-    email: email
     exec: exec
     halt: halt
     single: single
     suspend: suspend
     syslog: syslog
+    warn_once: warn_once
```

Comment:
I don't think a warn_once option should be supported. The recommendation should be syslog.

Reply:
> I don't think a warn_once option should be supported. The recommendation should be syslog.

I am just following the latest audisp-remote.conf man page

Reply:
@vojtapolasek remember that we need to provide sane security options not always all the options that a man page provides.